lopper/assists/xlnx_overlay_dt.py

```diff
@@ -138,6 +138,19 @@ def xlnx_generate_overlay_dt(tgt_node, sdt, options):
         sys.exit(1)        
 
     pl_node = None
+    has_valid_pl = None
```

Comment:
Hi @NavaKishoreManne ,
Please define it as False instead of None. As the value is set to True inside the below conditions

Reply:
Will fix in v2.